redis/src/cluster_async/mod.rs

```diff
@@ -928,15 +928,16 @@ where
         .await?;
 
         if let Some(duration) = topology_checks_interval {
-            let periodic_task = ClusterConnInner::periodic_topology_check(
-                connection.inner.clone(),
-                duration,
-                shutdown_flag,
-            );
+            let periodic_task =
+                ClusterConnInner::periodic_topology_check(connection.inner.clone(), duration);
             #[cfg(feature = "tokio-comp")]
-            tokio::spawn(periodic_task);
+            {
```

Comment:
really minor: is the {} is a linter requirment? 🤔

Reply:
Its a requirement for #[cfg]
`attributes on expressions are experimental see issue #15701 [https://github.com/rust-lang/rust/issues/15701](vscode-file://vscode-app/Applications/Visual%20Studio%20Code.app/Contents/Resources/app/out/vs/code/electron-sandbox/workbench/workbench.html) for more information`

Reply:
I agree that this is ugly